extensions/amp-consent/amp-consent.md

```diff
@@ -313,7 +313,7 @@ _Example_: Displays a prompt user interface
 
 AMP displays prompt UI on page load or by user interaction. The prompt UI is hidden based on the three user actions described below.
 
-AMP also supports external consent UI flow with the usage of `<amp-iframe>`. More information about the communication of user actions can be found [below](#prompt-actions-from-external-consent-ui).
+AMP also supports external consent UI flow via `promptUiSrc` which will load your custom iframe. More information about the communication of user actions can be found [below](#prompt-actions-from-external-consent-ui).
```

Comment:
Thanks! I like the general idea to migrate to `promptUiSrc`. 
Switching to `promptUiSrc` also requires sending the `ready` signal back to AMP. Let's make sure that's also documented.

Reply:
I added the ready signal documentation in `amp-consent.md`. Alternatively, I could mention it, and then link them to `integrating-consent.md` where the `ready` signal is already documentated.

Reply:
I prefer linking them to `integrating-consent.md` from this paragraph. We also cover the useful APIs like `enter-fullscreen` there. 
We can see us removing the session `##### Prompt Actions from External Consent UI` and direct readers to `#### Informing Consent response` section in the other doc. 
What do you think?

Reply:
Yup that sounds like a good idea